templates/cosmos_qa/templates.yaml

```diff
@@ -0,0 +1,352 @@
+dataset: cosmos_qa
+templates:
+  0293f8b7-20f8-472f-8861-6d2fea428441: !Template
+    id: 0293f8b7-20f8-472f-8861-6d2fea428441
+    jinja: '{{ context }}
+
+      Question: {{ question }}
+
+      Options:
+
+      A. {{ answer0 }}
+
+      B. {{ answer1 }}
+
+      C. {{ answer2 }}
+
+      D. {{ answer3 }}
+
+      ===
+
+      The best answer is
+
+      |||
+
+      {% if label == 0 %}
+
+      {{ answer0 }}
```

Comment:
For prompts where you label the answers (A. B. etc.) can you always have the model predict the label (A) instead of the answer? If we sometimes have it predict the label and sometimes the answer, it will get ambiguous training.

Reply:
Apologies for my carelessness. It has been fixed.